### DISCUSSION TITLE
Exclude specific extensions?

### DISCUSSION THREAD

**Discussion by pppaapp** [Difegue/LANraragi#785] [Q&A] [answered]
hi

I set the Calibre Library to Archive Directory.

The Calibre Library has a lot of epub, png, cbz, etc. files, which is very confusing.

I need an option to exclude files with specific extensions from the Archive Directory

**Comment by Difegue** [comment, marked as answer]
~~don't use calibre lol~~  

I believe you could fix this by putting the files you want in a subfolder of the calibre library, and then just use that as the content folder for LRR?  
Filtering specific extensions that we normally support is too weird an edge case, so I don't think I'd support that in the server itself. 🙇

**Reply by pppaapp** [reply]
That's sad news.
I needed this because the CBZ files I had were taking up too much space. 🥺

**Reply by pppaapp** [reply]
I solved it with a symbolic link...
